corehq/form_processor/models.py

```diff
@@ -258,9 +259,7 @@ class AbstractAttachment(models.Model):
 
     @property
     def filepath(self):
-        if getattr(settings, 'IS_TRAVIS', False):
-            return os.path.join('/home/travis/', str(self.attachment_id))
-        return os.path.join('/tmp/', str(self.attachment_id))
+        return os.path.join(gettempdir(), str(self.attachment_id))
```

Comment:
`TMPDIR` sounds like it could work. `/home/travis/tmp` didn't work cause i never created the `tmp` dir haha. very strange that the docs say it gets the first writeable dir. 

it was on the new setup